### DISCUSSION TITLE
Auto positioning to take floating header into account

### DISCUSSION THREAD

**Discussion by k1yoshida** [Akryum/v-tooltip#986] [Q&A]
hi! Thank you for all the work to create floating vue! 
I am using the v-tooltip directive on a website that has a floating header, and one issue I'm running into is when I use auto positioning for the popover, if the size of the popover still fits within the space above the element, it will appear behind the floating header. I**s there a way to add a custom value so if the tooltip will appear within the space defined by the header, it will automatically flip to the down position?** 

I know that floating vue uses Floating UI as for the popper positioning, and in their **documentation there is a "detectOverflow" option to "add padding"** to the boundary to detect for overflow. **Is this an option that is available for us to change within floating vue?**

**Comment by k-utsumi** [comment]
Maybe set boundary element. [REF](https://github.com/Akryum/floating-vue/discussions/378#discussioncomment-253523)
And set `overflowPadding`.

```
<VTooltip :boundary="wrapperElement" overflowPadding="16"> ... </VTooltip>
```

I tried to set the config while using nuxt, but encountered a problem. [REF](https://github.com/Akryum/floating-vue/issues/807#issuecomment-1801303632)
